package.json

```diff
@@ -339,13 +339,7 @@
 				"command": "lean.input.convert",
 				"key": "tab",
 				"mac": "tab",
-				"when": "editorTextFocus && editorLangId == lean && lean.input.isActive"
-			},
-			{
-				"command": "lean.input.convertWithNewline",
-				"key": "enter",
-				"mac": "enter",
-				"when": "editorTextFocus && editorLangId == lean && lean.input.isActive && !suggestWidgetVisible && !renameInputVisible && !inSnippetMode && !quickFixWidgetVisible && (!vim.active || vim.mode == 'Insert')"
+				"when": "editorTextFocus && !suggestWidgetVisible && editorLangId == lean && lean.input.isActive"
```

Comment:
Can you update the README to remove `lean.input.convertWithNewline` as well?

Maybe you could even add the GIF showing off the abbreviation from your PR comment.

Reply:
> Can you update the README to remove lean.input.convertWithNewline as well?

This is currently not mentioned in the readme - so there is nothing to remove.

> Maybe you could even add the GIF showing off the abbreviation from your PR comment.

I would wait until I experimented with the intellisense feature. Also, this GIF is not really meant to document the feature - too much is going on in it.

Reply:
Whoops, I thought I'd added all options to the README when I revamped it, but I guess I skipped this one. Sorry for the noise!